ref-farming/src/lib.rs

```diff
@@ -0,0 +1,218 @@
+/*!
+* Ref-Farming
+*
+* lib.rs is the main entry point.
+*/
+use near_sdk::borsh::{self, BorshDeserialize, BorshSerialize};
+use near_sdk::json_types::{ValidAccountId};
+use near_sdk::collections::{LookupMap, UnorderedMap};
+use near_sdk::{env, near_bindgen, Balance, AccountId, PanicOnDefault};
+
+use crate::farm::{Farm, FarmId};
+use crate::farm_seed::{FarmSeed, SeedId};
+use crate::farmer::Farmer;
+
+// for simulator test
+pub use crate::simple_farm::HRSimpleFarmTerms;
+pub use crate::view::FarmInfo;
+
+
+mod utils;
+mod errors;
+mod farmer;
+mod token_receiver;
+mod farm_seed;
+mod farm;
+mod simple_farm;
+mod storage_impl;
+
+mod actions_of_farm;
+mod actions_of_seed;
+mod actions_of_reward;
+mod view;
+
+/// sodu module is used to debug and testing,
+/// remove this module in release version
+mod sudo;
```

Comment:
let's add `#[cfg(build = "debug")]` though I'm not sure why is this needed for debugging either. If you have full access keys to the contract can in theory deploy new contract that will change owner if needed.

Reply:
I have removed this sudo mod, change it to owner.rs. What is this owner mod used for, has explained in next comment.